governance/working-group-chair-proposal-process.md

```diff
@@ -0,0 +1,82 @@
+# Environmental Sustainability TAG Working Group Chair Proposal Process
+
+TAG Environmental Sustainability Working Group Chairs operate as a team alongside the TAG's co-chairs and technical leads. 
+They provide guidance and directions to the working group members and ensure that the working group's efforts are in line with it's mission and values, as well as the overall goals of the TAG. They facilitate the working group's activities and coordinate its members' efforts. They schedule working group meetings, set meeting agenda, and maintain effective communication to ensure smooth collaboration among the working group's participants.
+Finally, they provide regular updates to the TAG, keeping the TAG leadership team and members informed about the group's progress and deliverables.
+
+## Caveats of Working Group Chairs
+
+* The position of a working group chair is not a paid position (TAG co-chairs and tech leads are also not paid positions)
+* Working group Chairs [have responsibilities](roles.md) to the Environmental Sustainability TAG, the leadership team, and the TOC
+* Nominations by the community, with endorsements by 1 or more working group/TAG co-chairs
```

Comment:
```suggestion
* Nominations made by the community, need endorsements by 1 or more working group/TAG co-chairs, or tech leads
```
Are tech leads allowed to provide endorsement?

Reply:
I dont see why not. +1 adding it

Reply:
+1 from me as well, approved suggested change for this.